src/main/java/de/leonhard/storage/Config.java

```diff
@@ -14,8 +14,6 @@
 @SuppressWarnings({"unused"})
 public class Config extends Yaml {
 
-  private List<String> header;
```

Comment:
Why is this not needed any more? Was the usage removed in the past, or why was it here? Do you know anything about that?

Reply:
There was 0 usage of this variable, with most references to headers living in `YamlEditor`.
Couldn't tell you the history of it, all I know is that it is no longer used.

Reply:
Ok then.